drivers/flash/flash_stm32_ospi.c

```diff
@@ -368,6 +368,7 @@ static int stm32_ospi_read_jedec_id(const struct device *dev)
 
 	cmd.Instruction = JESD216_CMD_READ_ID;
 	cmd.AddressSize = HAL_OSPI_ADDRESS_NONE;
+	cmd.AddressMode = HAL_OSPI_ADDRESS_NONE;
```

Comment:
correct : the AddressMode is of type OSPI_AddressMode
but could you please also check with `.AddressSize = stm32_ospi_hal_address_size(dev);`
because AddressSize = HAL_OSPI_ADDRESS_NONE;  is not correct  : AddressSize is of type **OSPI_AddressSize** 
-  HAL_OSPI_ADDRESS_8_BITS      
-  HAL_OSPI_ADDRESS_16_BITS    
-  HAL_OSPI_ADDRESS_24_BITS    
-  HAL_OSPI_ADDRESS_32_BITS

Reply:
Thanks, Also fixed now